release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -797,6 +797,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 You can attempt to resolve the issue with the steps in this link:https://kb.vmware.com/s/article/2002779[VMware KBase article]. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/5785341[[UPI vSphere\] Node scale-up doesn't work as expected]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918383[*BZ#1918383*])
 
+* An Open Virtual Network (OVN) bug causes persistent connectivity issues with Octavia load balancers. This problem affects Neutron subnets, or OpenShift namespaces, at random. Because of this bug, {product-title} 4.8 deployments that use Kuryr SDN are not recommended on {rh-openstack-first} 16.1 with OVN and OVN Octavia configured. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937392[*BZ#1937392*])
```

Comment:
Right, sorry for that, I might have phrased that in a confusing way initially, let me break it down a bit.

The problem is that when Octavia load balancers are created, OVN sometimes forgets to plug them into some Neutron subnets (at random). Effectively this means that the created LB might not be reachable from some of the subnets.

With "Neutron subnets, or OpenShift namespaces" I meant that when Kuryr is configured each OpenShift namespace has a separate subnet in Neutron. This means that if problem occurs the LB (implementing OpenShift Service) will not be reachable from OpenShift namespaces affected by the issue.

I'm not sure if it helps, happy to answer any questions.

Also do we want to add that it's not recommended only until that bug is fixed?